### ISSUE TITLE
Search results in "normal" (not compact) view missing bottom border

### ISSUE THREAD

**Issue by anarchivist** [sul-dlss/arclight#432]
![screen shot 2017-05-31 at 2 58 15 pm](https://cloud.githubusercontent.com/assets/73732/26655957/e6c83cf6-4611-11e7-8c11-76854f118630.png)

**Comment by mejackreed** [comment]
I thought that was by design :)

**Comment by jkeck** [comment]
@mejackreed this was my understanding too.  I believe @ggeisler did this intentionally.

**Comment by anarchivist** [comment]
Hm. Should we leave it until he comes back, then? The mocks are somewhat inconsistent:

**Collection level results only - all borders displayed**

![arclight-mockups-mvp](https://cloud.githubusercontent.com/assets/73732/26701476/286548f6-46d6-11e7-8973-e2093d905b37.jpg)

**All results - all borders displayed on collection-level results; no borders at all displayed on others** 
 
![arclight-mockups-mvp](https://cloud.githubusercontent.com/assets/73732/26701427/ebee3752-46d5-11e7-9707-daf306a8d550.jpg)

**Comment by drh-stanford** [comment]
i moved this out of ready and into design for discussion

**Comment by ggeisler** [comment]
@mejackreed and @jkeck are correct, omitting the bottom border is not a bug but intended (this is not the kind of thing I would miss!). @jvine and I previously discussed this and agree that it works visually. The goal is to clearly delineate each item but at the same time keep the page from feeling too boxy. Leaving the bottom of each item open helps led the user's eye from one result to the next. That's the goal, anyway I spent a fair amount of time experimenting and feel it works better than other options I tried.

The mockups are inconsistent, true, but that's because we never intended to produce pixel-perfect mockups that represent this level of visual design details. To do that would've required another week of design work, reconciling all of the minor differences in my and @jvine's pages, going back and updating all pages to reflect changes made in response to last-minute feedback, etc.
